test/fixtures/access-control/server/server.js

```diff
@@ -5,7 +5,7 @@
 
 var loopback = require('../../../..');
 var boot = require('loopback-boot');
-var app = module.exports = loopback({ localRegistry: true });
+var app = module.exports = loopback({ localRegistry: true, loadBuiltinModels: true });
```

Comment:
and why are tests failing without this... complaining that the builtin model `Application` is unknown?